src/test/groovy/org/mockito/release/notes/vcs/DefaultCommitApproverTest.groovy

```diff
@@ -0,0 +1,52 @@
+package org.mockito.release.notes.vcs
+
+import spock.lang.Specification
+
+class DefaultCommitApproverTest extends Specification {
+
+    def "should approve commit with sample message"() {
+        def approver = new DefaultCommitApprover()
+        def commit = new GitCommit("sampleId", "sample@email.com", "sampleAuthor", "sample commit message")
+
+        when:
+        def result = approver.isTrue(commit)
```

Comment:
Then I should remove `when` and `then` right? I always used to have `given`, `when`, `then` blocks but understand thats its a convention and I should replace `result`, `when:`, and `then:` with `expect:` right?